README.md

```diff
@@ -99,24 +99,32 @@ hard work - it is often underestimated how much effort is required.
    changes need to justify their existence with great value. Doubt about the
    necessity or value of a change will be a strong motivator for rejection.
 
-2. Be very pedantic and formal in describing any required language/compiler
+2. Showing example of real-world project that could benefit from an
+   improvement can greatly improve its chances of being accepted. D is intended
+   to be a practical language and a recurring questions when new
+   changes are considered is "does it really make real projects better?". If
+   that is not possible, try designing code examples with intention to answer
+   the very same question.
+
+3. The motivation section should showcase examples that currently are not
+   possible or are contorted in D, followed by their alternative implementation
+   using the proposed feature. At best, the example would be taken from existing
+   code (e.g. found in dub, phobos, etc). Next best is code that defines a
+   plausible artifact.   
+
+4. Be very pedantic and formal in describing any required language/compiler
    changes. For example, make separate sections for grammar changes, semantic
    changes, expected compiler error messages and so on. A DIP should provide
    a detailed design documentation for a compiler developer to implement it.
 
-3. Research any alternative approaches to solving the same problem and explain
+5. Stick to formal and technical language in general, avoid colloquialisms.  
```

Comment:
You may want to add: "A DIP should be formulated like a scientific paper, as opposed to an op-ed article."